runtime/jvmMain/src/kotlinx/benchmark/jvm/JvmBenchmarkRunner.kt

```diff
@@ -77,7 +78,12 @@ class JmhOutputFormat(private val reporter: BenchmarkProgress, private val suite
     }
 
     override fun endRun(result: Collection<RunResult>) {
-        reporter.endSuite(suiteName)
+        val summary = ByteArrayOutputStream().apply {
+            PrintStream(this, true, Charsets.UTF_8).use {
```

Comment:
This constructor was introduced in Java 10, but the library developement and jvmTarget version is Java 1.8.
```
e: /gradle-benchmarks/runtime/jvmMain/src/kotlinx/benchmark/jvm/JvmBenchmarkRunner.kt: (82, 13): None of the following functions can be called with the arguments supplied: 
public constructor PrintStream(p0: OutputStream!, p1: Boolean, p2: String!) defined in java.io.PrintStream
private constructor PrintStream(p0: Boolean, p1: OutputStream!, p2: Charset!) defined in java.io.PrintStream
private constructor PrintStream(p0: Boolean, p1: Charset!, p2: OutputStream!) defined in java.io.PrintStream
```
Please use `PrintStream(this, true, "UTF-8")` or `PrintStream(this, true, Charsets.UTF_8.toString())` instead.

Reply:
Fixed, yeah, forgot to switch JVM